tests/requirements-win.txt

```diff
@@ -1,3 +1,5 @@
+--extra-index-url https://pypi.anaconda.org/giumas/simple
+--extra-index-url https://pypi.anaconda.org/xoviat/simple
```

Comment:
Lxml was previously installed directly in appveyor.yml, per #1638. I had to create a repository because the lxml binaries for 3.6 were not in the original repository. During the process, I thought it would be cleaner to have the URLs in a requirements file so that it's not specific to windows.